contracts/multisig-prover/src/msg.rs

```diff
@@ -8,19 +8,38 @@ use crate::encoding::{Data, Encoder};
 
 #[cw_serde]
 pub struct InstantiateMsg {
+    // Address that can call UpdateWorkerSet
     pub admin_address: String,
+    // Address that can call UpdateSigning threshold, as well as automatically confirm worker set updates, bypassing verification
```

Comment:
```suggestion
    /// Address that can call all messages of unrestricted, admin and governance permission level, such as UpdateSigningThreshold. This address can execute messages that bypasses verification checks to rescue the contract if it got into an otherwise unrecoverable state due to external forces. It should match the address of the Cosmos governance module
```